Formula/idris2.rb

```diff
@@ -14,6 +14,7 @@ class Idris2 < Formula
   end
 
   depends_on "gmp" => :build
+  depends_on arch: :x86_64 # because of chezscheme
```

Comment:
#112065

Reply:
Because that other PR is marked “do not merge”, I'm merging this one, and the dependency can be removed in https://github.com/Homebrew/homebrew-core/pull/112065 of course.